index.html

```diff
@@ -53,5 +46,88 @@ <h3>I'am a software developer</h3>
             </div>
         </section>
     </div>
+
+    <section class="my-main-works">
+        <section class="recent-works">
+            <div class="works-title">
+                <h2>My Recent Works</h2>
+                <hr class="solid-line">
+            </div>
+        </section>
+        <section class="multi-project">
+            <div class="project-image"></div>
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div class="project-tag">
+                <p>CSS</p>
+                <p>html</p>
+                <p>bootstrap</p>
+                <p>Ruby</p>
+            </div>
```

Comment:
- [ ] As per, previous reviewer's suggestion kindly use `ul` and `li` tag while listing items. As it is the best way to list items in HTML.
Such As:
```suggestion
            <ul class="project-tag">
                <li>CSS</li>
                <li>html</li>
                <li>bootstrap</li>
                <li>Ruby</li>
            </ul>
```